tutorials/source-modeling/plot_source_alignment.py

```diff
@@ -176,6 +180,75 @@
                        src=src, subjects_dir=subjects_dir, dig=True,
                        surfaces=['head-dense', 'white'], coord_frame='meg')
 
+###############################################################################
+# Visualizing the transformation of digitized points
+# --------------------------------------------------
+# Let's step through the process that the transform is accomplishing.
+#
+# First, let's define a helper function to plot the alignment of the
+# points with the head.
+
+
+def plot_dig_alignment(points, coord_system='ras'):
+    points = points.copy()
+    if coord_system == 'mri':  # transform back to RAS
+        points = ((points - 128) * [1, -1, 1])[:, [0, 2, 1]]
+    else:  # m → mm
+        points *= 1e3
+    renderer = mne.viz.backends.renderer._get_renderer(
```

Comment:
```suggestion
    renderer = mne.viz.backends.renderer.create_3d_figure(
```

Reply:
@larsoner, this didn't work. `create_3d_figure` returns a scene and not the renderer.

Reply:
https://github.com/mne-tools/mne-python/blob/master/tutorials/source-modeling/plot_background_freesurfer_mne.py#L403 uses it

Reply:
@larsoner looks like you forgot to include the `scene=False` param